ansible/server-state-playbooks/ome-dundeeomero/playbook.yml

```diff
@@ -11,6 +11,27 @@
            ((ansible_virtualization_type is defined)
            and (ansible_virtualization_type == "VMware"))
 
+    # Not required for an OMERO server install - UoD specific implementation
```

Comment:
Fair enough, but that's certainly the point of the reviews is so that we express to the next person coming along here's how _we_ think it should be done.